base/types.go

```diff
@@ -726,6 +751,27 @@ type WrappedUprEvent struct {
 	Flags             WrappedUprEventFlag
 	ByteSliceGetter   func(uint64) ([]byte, error)
 	DecompressedValue []byte
+	TranslatedVB      *uint16 // nil if no vb translation
```

Comment:
Same here, recycle to be on the safer side. I think in dataObjPools.go
```
func (w *WrappedUprPool) Get() *base.WrappedUprEvent {
	wrappedUprEvent := w.pool.Get().(*base.WrappedUprEvent)
	// Ensure no residue
	wrappedUprEvent.UprEvent = nil
	wrappedUprEvent.ColNamespace = nil
	wrappedUprEvent.Flags = 0
	wrappedUprEvent.ByteSliceGetter = w.slicePool.GetByteSlice
	return wrappedUprEvent
}
```

Reply:
Done